locale/it/about/index.md

```diff
@@ -20,27 +24,39 @@ const server = http.createServer((req, res) => {
 });
 
 server.listen(port, hostname, () => {
-  console.log(`Server in esecuzione all'indirizzo http://${hostname}:${port}/`);
+  console.log(`Server running at http://${hostname}:${port}/`);
 });
 ```
-Ciò è in contrasto con il modello comune del giorno d'oggi, in cui vengono sfruttati i thread del sistema operativo. Il networking basato sui thread è piuttosto inefficiente e molto difficile da usare. Inoltre, gli utenti di Node sono liberi da preoccupazioni di processi che entrano in dead-lock, dal momento in cui non ci sono blocchi. Quasi nessuna funzione in Node esegue direttamente operazioni di I/O, perciò il processo non si blocca mai. Dal momento che nulla si blocca, è molto facile sviluppare sistemi scalabili in Node.
 
-Se qualcosa in questa terminologia ti è poco familiare, qui c'è un articolo completo su [Bloccante vs Non Bloccante][]. 
+Ciò è in contrasto con il modello di concorrenza più comune di oggi in cui vengono utilizzati 
+i thread. Il networking basato su thread è relativamente inefficiente e 
+molto difficile da utilizzare. Inoltre, gli utenti di Node sono liberi da preoccupazioni di 
+blocco permanente del processo, poiché non ci sono blocchi. Quasi nessuna funzione in Node 
+esegue direttamente I / O, quindi il processo non blocca mai. Perché nulla blocca, i sistemi 
+scalabili sono molto ragionevoli da sviluppare con Node.js.
+
+
+
+Se alcuni dei termini utilizzati non ti sono familiari, ecco un articolo completo
+ (in inglese) [Bloccante vs Non Bloccante][].
 
 ---
 
-Node è simile in design, e influenzato, da sistemi come la [Event Machine][] di Ruby o [Twisted][] di Python. Node porta il modello ad eventi un po' oltre. Presenta un [loop di eventi][] come un costrutto a runtime invece che come libreria. In altri sistemi c'è sempre una chiamata bloccante per chiamare un Event Loop.
+Node è  influenzato da sistemi come Ru Ruby's
```

Comment:
Typo "Ru". The original translation sounds better. "da sistemi come la [Event Machine][] di Ruby o [Twisted][] di Python"